### ISSUE TITLE
How to out put captured output on the UI？

### ISSUE THREAD

**Issue by LukeHe** [chovanecm/sacredboard#80]

**Comment by chovanecm** [comment]
It is already there when you click on an experiment, a detail view should open. Do you have troubles with it?

**Comment by LukeHe** [comment]
@chovanecm , thank you! I know that, but I want to out put the keras output of model.fit function, how I can do that?

**Comment by LukeHe** [comment]
By the way, does sacredboard support the latest tensorboard?

**Comment by chovanecm** [comment]
According to the documenation of Keras, `model.fit` returns
> A History object. Its History.history attribute is a record of training loss values and metrics values at successive epochs, as well as validation loss values and validation metrics values (if applicable).

This sounds like a job for the Sacred [Metrics API](http://sacred.readthedocs.io/en/latest/collected_information.html#metrics-api) that Sacredboard can display. However, you have to find by yourself how to iterate over the `History` object returned by Keras to add it to the Sacred DB.

**Comment by chovanecm** [comment]
> By the way, does sacredboard support the latest tensorboard?

No as I haven't found time for Sacredboard since I started my current job yet :-(
But I am already considering steps that would allow me to spend some time on the project again.

**Comment by chovanecm** [comment]
Hi again. I have fixed running Tensorboard in the latest dev version. But it always starts it on port 6006, so no multiple tensorboards supported (yet) :(
